scenarioo-docu-generation-example/src/test/java/org/scenarioo/uitest/example/config/ExampleUITestDocuGenerationConfig.java

```diff
@@ -46,6 +46,7 @@
 	 * (e.g. "trunk" or "bugfix-branch-2013-11")
 	 */
 	public static final String EXAMPLE_BRANCH_NAME = "wikipedia-docu-example";
+	public static final String EXAMPLE_BRANCH_NAME_DEV = "wikipedia-docu-example-dev";
```

Comment:
Make two names like "wikipedia-example-version-1.0" and "wikipedia-example-version-1.1" or "wikipedia-example-develop".